src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -1,105 +1,206 @@
+        :status-indicator? false}]]
+
+     :multiuser
+     [preview-list/preview-list {:type :user :size 20}
+      users]
+
+     :multinetwork ;; TODO: use multinetwork preview-list type when available
+     [preview-list/preview-list {:type :user :size 20}
+      (map #(hash-map :profile-picture %) networks)]
+
+     :audio
+     [tag-skeleton {:theme theme :text (str duration)}
+      [rn/view {:style style/audio-tag-icon-container}
+       [icons/icon :i/play {:color style/audio-tag-icon-color :size 12}]]]
+
+     :group
+     [tag-skeleton {:theme theme :size size :text group-name}
+      [group-avatar/group-avatar
+       {:icon-name :i/members
+        :size      (if (= size 24) :small 28)
+        :color     (colors/custom-color :purple 50)}]]
+
+     (:channel :community)
+     [communities-tag (assoc props :channel? (= type :channel))]
+
+     :token
+     [tag-skeleton {:theme theme :size size :text (str amount " " token-name)}
+      [rn/image {:style (style/circle-logo size) :source token-logo}]]
+
+     :network
+     [tag-skeleton {:theme theme :size size :text network-name}
+      [rn/image {:style (style/circle-logo size) :source network-logo}]]
+
+     :collectible
+     [tag-skeleton
+      {:theme theme
+       :size  size
+       :text  (str collectible-name " #" collectible-number)}
+      [rn/image {:style (style/rounded-logo size) :source collectible}]]
+
+     :account
+     [tag-skeleton {:theme theme :size size :text account-name}
+      [account-avatar/view {:emoji emoji :size (if (= size 24) 20 28)}]]
+
+     :address
+     [address-tag props]
+
+     :icon
+     [icon-tag props]
+
+     nil)])
+
+(def view
+  "Properties:
+  type, state, blur? & customization-color
+
+  Depending on the `type` selected, different properties are accepted:
+  - `:default` or `nil`:
+    - size
+    - profile-picture
+    - full-name
+
+  - `:multiuser`:
+    - users (vector of {:profile-picture pic, :full-name \"a name\"})
+
+  - `:group`
+    - size
+    - group-name
+
+  - `:community`
+      - size
+      - community-logo (valid rn/image :source value)
+      - community-name
+
+  - `:channel`
+    - size
+    - community-logo (valid rn/image :source value)
+    - community-name
+    - channel-name
+
+  - `:token`
+    - size
+    - token-logo (valid rn/image :source value)
+    - amount
+    - token-name
+
+  - `:network`
+    - size
+    - network-logo (valid rn/image :source value)
+    - network-name
+
+  - `:multinetworks`
+    - networks (vector of {:network-logo pic, :network-name \"a name\"})
+
+  - `:account`
+    - size
+    - account-name
+    - emoji (string containing an emoji)
+
+  - `:collectible`
+    - size
+    - collectible (valid rn/image :source value)
+    - collectible-name
+    - collectible-number
+
+  - `:address`
+    - size
+    - address (string)
+
+  - `:icon`
+    - size
+    - icon
+    - context (string)
+
+  - `:audio`
+    - duration (string)
+  "
```

Comment:
Long docstring :cry:  but in figma, this component has many variants

Reply:
A separate appreciation for the docstring :clap: :clap: :clap: